.kiro/specs/tls-rustls-migration/design.md

```
@@ -2,218 +2,464 @@
+            },
         }
-        if let Some(key_path) = &self.client_key_path {
-            ssl_opts = ssl_opts.with_client_key_path(Some(key_path.clone()));
-        }
-        // Apply verification flags if supported by SslOpts API
-        ssl_opts = ssl_opts.with_danger_accept_invalid_certs(!self.verify_peer);
-        ssl_opts = ssl_opts.with_danger_skip_domain_validation(!self.verify_hostname);
-        Some(ssl_opts)
+
+        let client_config = client_config_builder.with_no_client_auth();
+
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)